bin/sdk/gutenberg.js

```diff
@@ -4,20 +4,50 @@
  */
 const fs = require( 'fs' );
 const path = require( 'path' );
+const reduce = require( 'lodash/reduce' );
+const pickBy = require( 'lodash/pickBy' );
+const isEmpty = require( 'lodash/isEmpty' );
 
 exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 	const baseConfig = getBaseConfig( {
 		cssFilename: '[name].css',
 		externalizeWordPressPackages: true,
 	} );
-	const editorScript = path.join( inputDir, 'editor.js' );
-	const viewScript = path.join( inputDir, 'view.js' );
+
+	const presetPath = path.join( inputDir, 'index.json' );
+	const utilsPath = path.join( inputDir, 'utils' );
+
+	let viewBlocksPoints;
+	let viewScriptEntry;
+	if ( fs.existsSync( presetPath ) ) {
+		const presetBlocks = require( presetPath );
+
+	  // Helps split up each block into its own folder view script
+		viewBlocksPoints = reduce( presetBlocks, (obj, block) => ( { [ block + '/view'] : [path.join( inputDir, '../../'+ block +'/view.js' )]  } )  );
```

Comment:
What is this `reduce` supposed to do? `obj` is never used, so the whole call will return the returned value for the last block?

Maybe you want this?
```js
fromPairs( map(
  blocks,
  block => [ block + '/view', path.join( ..., block, 'view.js' ) ]
) );
```